### ISSUE TITLE
[MU4 Task] Implement "System", "Staff" and "Capo" text lines in palettes (with corresponding Properties settings)

### ISSUE THREAD

**Issue by bkunda** [musescore/MuseScore#10098]
**Task description**
"System text line" and "Staff text line" palette objects, together with their editable settings in **Properties**, have not yet been brought over to MS4. 

These are the objects from MS3:

<img width="1149" alt="Screenshot 2021-12-15 at 9 43 14 pm" src="https://user-images.githubusercontent.com/86290556/146172173-59955652-f4db-4a7f-b768-9d8927832bfb.png">

This task proposes to reinstate these objects in palettes (under "Lines"), with the following settings available in **Properties**:

<img width="858" alt="Screenshot 2021-12-15 at 9 44 54 pm" src="https://user-images.githubusercontent.com/86290556/146172426-34f55d0a-6919-425a-b914-fddda72261a5.png">

In addition, a third text line object "Capo text line" should also be introduced, triggering the same settings as above but with the Roman numeral "VII" entered as default beginning text:

<img width="859" alt="Screenshot 2021-12-15 at 9 45 58 pm" src="https://user-images.githubusercontent.com/86290556/146172597-4bda0d0a-b333-4fe9-ab1b-a5dfad58151c.png">

In summary, all three objects should be visible in palettes (currently, only the Roman numeral line object is visible in MS4):

<img width="1390" alt="Screenshot 2021-12-15 at 9 46 23 pm" src="https://user-images.githubusercontent.com/86290556/146172721-3b74e74a-2622-4ad2-8aeb-d1c77dc74158.png">

**Figma design file**
https://www.figma.com/file/l0TQUrNFZmEfrZnfHhFDfb/Musescore-Notation-App-MASTER?node-id=20952%3A676602

This task should resolve #9878 

Let me know if there are any questions.
Thanks!

**Comment by RomanPudashkin** [comment]
I'm working on it

**Comment by DmitryArefiev** [comment]
Hi @bkunda, Can you check this on Windows and Mac please? (#10240)

Thanks!

**Comment by bkunda** [comment]
Almost perfect from my end :-) 

One issue is that system text line properties don't update across parts. The actual text does__, but the line style (specifically dashed line and hook properties) aren't synchronising across parts. 

https://user-images.githubusercontent.com/86290556/149282319-5e8214b0-c14e-40e4-a181-cb3806c2f768.mp4

**Comment by RomanPudashkin** [comment]
@bkunda unfortunately, we have the same bug in MU3

https://user-images.githubusercontent.com/67867444/149405554-c25683a0-1fd4-4540-8839-59f7120c15b6.mp4

It is better to create a separate issue for this bug

**Comment by bkunda** [comment]
> addition

Ok will do!

**Comment by DmitryArefiev** [comment]
Tested LinuxMint 20.2 - FIXED
